pkgs/by-name/oi/oink-solver/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  lace,
+  boost,
+  build-extra-tools ? false,
+  doCheck ? false,
+}:
+
+stdenv.mkDerivation rec {
+  inherit doCheck;
+
+  pname = "oink";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "trolando";
+    repo = pname;
+    rev = "257d0d088361204449c9bb204b457c2042e26cae";
```

Comment:
Is there no tagged release?

Reply:
I'd make an issue upstream asking them to tag (future) releases, as it makes it easier for downstream packaging.

If they don't tag their release in the near future, add a comment linking to that upstream issue in the code, explaining that they haven't tagged a release.

Reply:
We'll see how it goes: https://github.com/trolando/oink/issues/23